kubernetes/helm_charts/secor/config/secor.common.properties

```diff
@@ -357,7 +357,8 @@ secor.max.message.size.bytes=100000
 
 # Class that will manage uploads. Default is to use the hadoop
 # interface to S3.
-secor.upload.manager.class=com.pinterest.secor.uploader.AzureUploadManager
+# secor.upload.manager.class=com.pinterest.secor.uploader.AzureUploadManager
```

Comment:
@ddevadat Don't we have to use CSP specific upload manager? It looks like you have changed it from AzureUploadManager to S3UploadManager.

Reply:
secor doesnt have an oci specific upload manager. OCI object storage is s3 api compliant. This means OCI can use S3UploadManager